docs/components/processor.rst

```diff
@@ -34,6 +34,114 @@ Then you can start and stop the processor and tweak files and all that jazz.
    The stackwalk binaries are in ``/stackwalk`` in the container.
 
 
+Processing crashes
+==================
+
+Running the processor is pretty uninteresting since it'll just sit there until
+you give it something to process.
+
+In order to process something, you first need to acquire raw crash data, put the
+data in the S3 container in the appropriate place, then you need to add the
+crash id to the "socorro.normal" RabbitMQ queue.
+
+We have helper scripts for these steps.
+
+
+``scripts/fetch_crash_data.py``
+-------------------------------
+
+This will fetch raw crash data from -prod and save it in the appropriate
+directory structure rooted at outputdir.
+
+Usage from host::
+
+  $ docker/as_me.sh scripts/fetch_crash_data.py <outputdir> <crashid> [<crashid> ...]
+
+
+For example (assumes this crash exists)::
+
+  $ docker/as_me.sh scripts/fetch_crash_data.py ./testdata 5c9cecba-75dc-435f-b9d0-289a50170818
+
```

Comment:
I think "Why would I want to mess with the directory?" unpacks into two questions:

1. why would I want to have the uid/gid for those files be my user account on my host?
2. why support multiple directories and not just a single one?

Towards the first one, if the uid/gid isn't the user account on your host, then in order to manipulate files in that directory from the host, you have to use sudo and that gets real old real fast. Adding crashes, removing crashes, adjusting raw crash data, and so on.

Towards supporting multiple directories, this makes it much easier to support working on multiple things at the same time. It lets you group crashes for a specific work project. Then you can process them and manipulate them and all that separate from other projects. Also, you can delete them when you're done that project which is pretty important.

I'm not sure I want to have it default to a specific directory because I really want crashes associated with the project/bug that they're being used for so they can be deleted asap. If you want that anyways, I think that's better solved by you having a shell script covering your specific work flow on your machine.